design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -122,7 +122,8 @@ type Mutation {
 
     addRequisitionListItemToCart(
         requisition_list_id: ID!, @doc(description: "unique Id of requisition list")
-        item_ids: [ID!]! @doc(description: "selected requisition list items that are to be added")
+        item_ids: [ID!]!, @doc(description: "selected requisition list items that are to be added")
+	is_replace: Boolean @doc(description: "replaces or merges the existing cart items with requisition list items")
```

Comment:
I'd prefer a separate mutation for this functionality. The name `addRequisitionListItemToCart` doesn't describe this separate behavior (destroying existing cart items). I'd like to see a mutation that makes it more clear in the name that it's destructive